### DISCUSSION TITLE
Proxmox und Raspberrymatic

### DISCUSSION THREAD

**Discussion by BobRoss88** [OpenCCU/OpenCCU#2518] [🇩🇪 Fragen&Antworten]
Schönen guten Tag jemand das selbe Problem das die Raspberrymatic unter Proxmox keine Geräte mehr findet? Es lief bestimmt 6 Monate ohne Probleme. Heute habe ich Proxmox neuinstalliert weil sich mit der zeit einfach zuviel Mist angesammelt hat. 
Habe im Moment einen leeren proxmoxserver mit RaspberryMatic-3.71.12.20231020-ova installiert und ich kann machen was ich will, ich kann keine geräte anlernen. HmIP-RFUSB 000420C9A18382:0 habe ich verbunden und Duty Cycle	und Carrier Sense zeigt es mir ganz normal an wie vorher.

**Comment by Baxxy13** [comment]
[Link](https://homematic-forum.de/forum/viewtopic.php?f=31&t=79828&hilit=problemkind)

Da ist auch ein Link drin wie man gucken kann ob der HmIP-RFUSB überhaupt was empfängt.

**Reply by BobRoss88** [reply]
Danke ich geh aber davon aus, dass er sendet. hab jetzt Proxmox dreimal neu aufgesetzt dreimal raspberrymatic in einer vm am laufen gehabt, leider keine geräte gefunden. In home assistant über das raspberrymatic addon findet er geräte. Aber egal dann lass ich es in home assistant laufen
